### PR TITLE
Updates to SPV_GOOGLE_user_type

### PR DESCRIPTION
**Pull request by sudonatalie**

### CODE REVIEW

**Comment by sudonatalie** [comment]
@chaoticbob Please take a look

File: extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc [hunk 1]
```diff
@@ -18,6 +18,7 @@ Contributors
 - Hai Nguyen, Google
 - David Neto, Google
 - Ehsan Nasiri, Google
+- Natalie Chouinard, Google
 
 Status
 ------
```
Comment (dgkoch) [inline extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc:24]: Version table below this needs to be updated.

Comment (sudonatalie) [reply extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc:24]: Do you mean the Revision History table at the bottom of this file? I added a row to it.

Comment (dgkoch) [reply extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc:24]: No, I mean lines 32 and 33, which are collapsed in the diff and I can't comment on them. 
They should be updated to match the latest entry in the Revision History table.

Comment (sudonatalie) [reply extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc:24]: Oops, don't know how I missed that on the second look too. Thanks!

File: extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc [hunk 2]
```diff
@@ -28,8 +29,8 @@ Version
 
 [width="40%",cols="25,25"]
 |========================================
-| Last Modified Date | 2019-04-22
-| Initial            | 1
+| Last Modified Date | 2024-01-25
+| Revision           | 2
```
Comment (dgkoch) [inline extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc:33]:
```suggestion
| Revision           | 5
```

File: extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc [hunk 3]
```diff
@@ -45,13 +46,13 @@ extension.
 
 Overview
 --------
-This extension provides one new decoration, UserTypeGOOGLE, that allows HLL
+This extension provides one new decoration, UserTypeGOOGLE, that allows HLSL
```
Comment (dgkoch) [inline extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc:49]: I suspect this is actually supposed to be HLL for "High level languages"  (This isn't only useful for HLSL is it?)

Comment (sudonatalie) [reply extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc]: Makes sense. I think it's only used for HLSL right now (the resource types below are HLSL specific), but it wouldn't have to be limited to that.

**Review by dgkoch — APPROVED** [review]
LGTM now.

**Comment by dgkoch** [comment]
> Thanks @dgkoch, and sorry for the excess back and forth on those changes! Can you merge this PR for me?

I would prefer to leave that to @bashbaug

**Review by s-perron — APPROVED** [review]

**Pull request merged**